release_docs/USING_HDF5_CMake.txt

```diff
@@ -234,7 +234,7 @@ adjust the forward slash to double backslashes, except for the HDF_DIR
 environment variable.
 
 NOTE: this file is available in the HDF5 repository, for more information see:
-    https://github.com/HDFGroup/hdf5/blob/develop/release_docs/USING_CMake_Examples.txt
+    https://github.com/HDFGroup/hdf5/blob/v1_14/release_docs/USING_CMake_Examples.txt
```

Comment:
Dead link

Reply:
@bmribler, should it be develop?

Reply:
or hdf5_1_14, which is will not  be a dead link?

Reply:
should be in 1.14

Reply:
Check in hdf5 repo for it.  Maybe URL needs to be edited.